app/src/main/java/com/example/userinfo/ChangeView.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.userinfo
+
+import android.os.Binder
+import android.view.View
+import android.widget.Button
+import androidx.constraintlayout.widget.Group
+import com.example.userinfo.databinding.ActivityUserInputBinding
+
+class ChangeView {
+     fun changeVisibilities(
+         visible: Boolean,
```

Comment:
There is nothing wrong here, but `visible` is not the wrong parameter right, 
Because `visible` can not decide to enable a field or not.
Correct name would be like `editable` or something which tells the state.